vars/buildPlugin.groovy

```diff
@@ -15,15 +15,19 @@ def call(Map addonParams = [:])
 		'osx-x86_64': 'osx64',
 		'ubuntu-ppa': 'linux',
 		'windows-i686': 'windows/win32',
-		'windows-x86_64': 'windows/x64'
+		'windows-x86_64': 'windows/x64',
+		'windows-i686-uwp': 'windows/win32-uwp',
+		'windows-x86_64-uwp': 'windows/x64-uwp'
 	]
 	def PLATFORMS_DEPLOY = [
 		'android-armv7',
 		'android-aarch64',
 		'osx-x86_64',
 		'ubuntu-ppa',
 		'windows-i686',
-		'windows-x86_64'
+		'windows-x86_64',
+		'windows-i686-uwp',
+		'windows-x86_64-uwp'
```

Comment:
Are you aware of the implications if `windows-i686-uwp` and `windows-x86_64-uwp` are added to the `PLATFORMS_DEPLOY` list?

Reply:
Was a little exuberant :smile:. Becomes removed.

Is it also as zip without available on e.g. https://jenkins.kodi.tv/blue/organizations/jenkins/xbmc%2Fvisualization.projectm/detail/PR-39/18/artifacts ?

Reply:
Forget the question the Link https://jenkins.kodi.tv/job/xbmc/job/visualization.projectm/job/PR-39/18/artifact/cmake/addons/build/zips/visualization.projectm+android-aarch64/visualization.projectm-3.0.0.zip is given to Kodi's build folder.